src/actions/auction/NoLossCollateralAuctionSPTActions.sol

```diff
@@ -104,7 +104,7 @@ contract NoLossCollateralAuctionSPTActions is NoLossCollateralAuctionActionsBase
 
         // approve the Sense Finance Adapter to transfer `target` tokens on behalf of the proxy
         if (redeemParams.approveTarget != 0) {
-            IERC20(redeemParams.target).approve(redeemParams.adapter, targetAmount);
+            IERC20(redeemParams.target).safeApprove(redeemParams.adapter, targetAmount);
```

Comment:
why test for redeemParams.approveTarget but safeApprove targetAmount? we should do the same here as in the VaultSPTActions (user may chose custom redeemParams.approveTarget so need to safeApprove with this amount) @SolcAndMe @amarcu

Reply:
Updated.